consensus/raft/config.go

```diff
@@ -48,6 +51,11 @@ type Config struct {
 	CommitRetries int
 	// How long to wait between retries
 	CommitRetryDelay time.Duration
+	// InitPeerset provides the list of initial cluster peers for new
+	// peers (with no prior state). It is when Raft was already initialized
+	// or when when starting in staging mode, before adding the peer
```

Comment:
typo "when when".  Also I don't understand the last sentence.  What 'is when Raft was already initialized?'